api/tables/store.js

```diff
@@ -49,22 +49,26 @@ export function createStoreTable (region, tableName, options = {}) {
     /**
      * Bind a link CID to an account
      *
+     * @typedef {import('../service/types').StoreItemOutput} StoreItemOutput
+     * 
      * @param {import('../service/types').StoreItemInput} item
+     * @returns {Promise<StoreItemOutput>}
      */
-    insert: async ({ uploaderDID, link, proof, origin, size = 0 }) => {
+    insert: async ({ space, link, origin, size = 0, issuer, invocation }) => {
+      /** @type StoreItemOutput */
       const item = {
-        uploaderDID,
-        payloadCID: link,
-        applicationDID: '',
-        origin: origin || '',
+        space,
+        link: link.toString(),
         size,
-        proof,
-        uploadedAt: new Date().toISOString(),
+        origin: origin?.toString(),
+        issuer,
+        invocation: invocation.toString(),
+        insertedAt: new Date().toISOString(),
       }
 
       const cmd = new PutItemCommand({
         TableName: tableName,
-        Item: marshall(item),
+        Item: marshall(item, { removeUndefinedValues: true }),
```

Comment:
This lets us send in `shard: undefined` and `marshall` will remove that property for us.